tasks/RedHat/generic/configure-selinux.yml

```diff
@@ -4,9 +4,11 @@
 - debug:
     msg: "imported RedHat/generic/configure-selinux.yml"
 
-- name: Disable SELinux
+- name: Set SELinux State
   selinux:
-    state: disabled
+    state: "{{ sap_preconfigure_selinux_state }}"
     policy: targeted
+  register: selinux_output
+  notify: "Reboot Required handler"
```

Comment:
reboot is required for all selinux state changes?

Reply:
Thanks! I tested again and found that a state change is not sufficient to determine if a reboot is required. For example:
1) sestatus before: permissive. mode from config file: permissive
sap_preconfigure_selinux_state=disabled -> state change -> reboot required (OK)
2) sestatus before: permissive. mode from config file: permissive
sap_preconfigure_selinux_state=disabled -> no state change -> no reboot required (not OK)
I should probably evaluate the return value of reboot_required, like in the selinux role.

Reply:
solved in https://github.com/berndfinger/sap-preconfigure/commit/2e2dcbcf99e13ce0a936d182061d34a5d05cbbe2